### DISCUSSION TITLE
One-option slider / disabled slider

### DISCUSSION THREAD

**Discussion by timvandam** [mantinedev/mantine#685] [Feedback]
I am using a slider to allow users to select some number in some range. However, in my case this range can also be 0 (as in, no numbers can be selected except 0). In such cases I think it would be great to be able to disable the slider so the user does not think that the slider is broken. Of course this is already possible visually by changing the color and by disabling labels, but I think a native solution makes sense since other input components have them too.

Aside from this, perhaps it is an idea to center the thumb in the case where only one value can be selected. It is currently all the way on the left: 
![image](https://user-images.githubusercontent.com/35376389/149675799-d8c53e0f-391d-4a6d-95d6-bd15e056b60e.png)

**Comment by rtivital** [comment]
Thanks for feedback, we will consider adding disabled state to the slider
